src/crafting.cpp

```diff
@@ -74,7 +74,7 @@ static const trait_id trait_BURROW( "BURROW" );
 static bool crafting_allowed( const player &p, const recipe &rec )
 {
     if( p.morale_crafting_speed_multiplier( rec ) <= 0.0f ) {
-        add_msg( m_info, _( "Your morale is too low to craft..." ) );
+        add_msg( m_info, _( "Your morale is too low to craft such a hard thing..." ) );
```

Comment:
`hard` -> `difficult` might be a better word choice? Since `hard` might mean something that is hard rather than soft.